be/src/olap/push_handler.cpp

```diff
@@ -382,6 +405,52 @@ void PushBrokerReader::print_profile() {
     LOG(INFO) << ss.str();
 }
 
+Status PushBrokerReader::_get_next_reader() {
+    while (true) {
```

Comment:
why need while true here?

Reply:
Deleted. I imitated the `vfile_scanner.cpp` before.